.github/workflows/ci.yaml

```diff
@@ -55,12 +56,16 @@ jobs:
       #       ${{ github.workflow }}-rust-${{ matrix.os }}-
       - name: Decrypt test keys
         run: openssl aes-256-cbc -d -K ${{ secrets.OPENSSL_KEY }} -iv ${{ secrets.OPENSSL_IV }} -in tests/testkeys/iak-dev.pem.enc -out tests/testkeys/iak-dev.pem
-      - name: Cargo test (with one retry on failure)
-        uses: nick-invision/retry@v2.4.0
+      # Generate the Cargo.lock that will be used for the tests so we can upload it as an artifact
+      - name: Generate lockfile
+        run: cargo generate-lockfile
+      - name: Upload Cargo.lock
+        uses: actions/upload-artifact@v2
         with:
-          timeout_minutes: 30
-          max_attempts: 2
```

Comment:
Don't need the timeout or max attempts any more?

Reply:
We don't need to retry at all anymore now that we fixed ironcore-id